ArjunaJTS/jtax/classes/com/arjuna/ats/internal/jta/resources/jts/orbspecific/XAResourceRecord.java

```diff
@@ -397,27 +397,25 @@ public void rollback() throws org.omg.CORBA.SystemException,
 						case XAException.XA_RBTIMEOUT:
 							destroyState();
 							break;
-						case XAException.XAER_NOTA:
```

Comment:
I don't think it is this  because NOTA actually goes down a different route due to https://github.com/jbosstm/narayana/pull/2272/files#diff-c0091e065ebb4dea411f68b3ea81f32d4b0ebe2b1349c2876073d576fdc505b8R354

The XAER_NOTA seems like it should not be in here therefore.